packages/browser-wallet/src/popup/pages/IdProofRequest/DisplayStatement/DisplayStatement.tsx

```diff
@@ -82,8 +74,32 @@ function StatementTooltip({ reveal }: StatementTooltipProps) {
                 </Button>
             }
         >
-            <h3>{header}</h3>
-            <div className="white-space-break">{body}</div>
+            <h3>
+                {reveal ? (
+                    <Trans
+                        ns="idProofRequest"
+                        i18nKey="displayStatement.revealTooltip.header"
+                        components={{ 1: <WarningTooltipIcon className="display-statement__tooltip-icon" /> }}
+                    />
+                ) : (
+                    t('secretTooltip.header')
+                )}
+            </h3>
+            <div className="white-space-break">
+                {reveal ? (
+                    <Trans
+                        ns="idProofRequest"
+                        i18nKey="displayStatement.revealTooltip.body"
+                        components={{ 1: <ExternalLink path={urls.zkpDocumentation} /> }}
+                    />
+                ) : (
+                    <Trans
+                        ns="idProofRequest"
+                        i18nKey="displayStatement.secretTooltip.body"
+                        components={{ 1: <ExternalLink path={urls.zkpDocumentation} /> }}
+                    />
+                )}
```

Comment:
is there a reason you can't do this?:
```suggestion
                    <Trans
                        ns="idProofRequest"
                        i18nKey={reveal ? 'displayStatement.revealTooltip.body' : 'displayStatement.secretTooltip.body'}
                        components={{ 1: <ExternalLink path={urls.zkpDocumentation} /> }}
                    />
```